build/Build.cs

```diff
@@ -300,10 +304,8 @@ void ErrorLogger(OutputType type, string text)
           .Requires(() => !IsModule)
           .Executes(() =>
           {
-              //dotnet %userprofile%\.nuget\packages\swashbuckle.aspnetcore.cli\4.0.1\lib\netcoreapp2.0\dotnet-swagger.dll tofile --output swagger.json bin/Debug/netcoreapp2.2/VirtoCommerce.Platform.Web.dll VirtoCommerce.Platform
-              //better use in the future a .Net Global Tool https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/master/README-v5.md#retrieve-swagger-directly-from-a-startup-assembly
-              var swashbucklePackage = NuGetPackageResolver.GetGlobalInstalledPackage("swashbuckle.aspnetcore.cli", "4.0.1", "dotnet-swagger.dll");
-              var swashbucklePath = swashbucklePackage.Directory / "lib" / "netcoreapp2.0" / "dotnet-swagger.dll";
+              var swashbucklePackage = NuGetPackageResolver.GetGlobalInstalledPackage("swashbuckle.aspnetcore.cli", "5.0.0", "dotnet-swagger.dll");
+              var swashbucklePath = swashbucklePackage.Directory / "tools" / "netcoreapp3.0" / "any" / "dotnet-swagger.dll";
```

Comment:
It's not good to hard-code paths like "netcoreapp3.0".  It should be better to search for dll.

Reply:
@krankenbro Will this comment be addressed?

Reply:
@yecli fixed